packages/aws-cdk-lib/jest.config.js

```diff
@@ -1,12 +1,19 @@
 const baseConfig = require('@aws-cdk/cdk-build-tools/config/jest.config');
 
+/** @type {import('ts-jest').JestConfigWithTsJest} */
 module.exports = {
```

Comment:
So there definitely is speedup here, but since we exclude the test files from build we would be relying on users to make sure TS compiles are good for test files with their editor tooling. Which most likely is okay?

Reply:
It will blow up for them when they run the build command locally or on the PR I guess.
But if we are giving on type checking, can just switch to swc already?

Reply:
Either way, using ts-jest would also allow us to change the developer workflow to use `yarn tsc --watch --noEmit` which is significantly faster.